website/www/site/content/en/blog/beamquest.md

```diff
@@ -0,0 +1,37 @@
+---
+title:  "Get Started with Apache Beam: An Open Source Quest sponsored by Google Cloud"
+date:   2023-06-09 00:00:01 -0800
+categories:
+  - blog
+aliases:
+  - /blog/2023/06/09/beam-quest.html
+authors:
+  - svetaksundhar
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### **What is the quest?**
+[Get Started with Apache Beam Quest](https://www.cloudskillsboost.google/quests/310) is a series of four qwiklabs that test proficiency in Apache Beam. Upon completion, a sharable credential badge is granted!
+
+
+### **Apache Beam Playground Features**
+
+We’re excited to announce the release of the “Get Started with Apache Beam” quest, a series of four online labs that venture into different Apache Beam concepts. When you complete all four labs, you’ll earn a [Google Cloud badge](https://www.cloudskillsboost.google/quests/310) that you can share on platforms like LinkedIn. Earning this badge should take less than seven hours total, and signing up for the quest costs $20 (however there are often free specials for people who attend Beam events, such as [Meetups](https://www.meetup.com/topics/apache-beam/), [Beam Summit](https://beamsummit.org/), and [Beam College](https://beamcollege.dev/)).
+
+Beam is one of the largest big data open source projects actively being developed. Over the past six years, the Apache Beam community has seen tremendous growth in the number  of contributors, committers, and users. If you’re a long time Beam user, you can now earn a badge to show your skills to potential employers. If you’re new to Beam, you can begin your learning journey with this quest. To attempt this quest, you don’t need any prior knowledge of data processing or distributed systems. All you need is elementary knowledge about programming.
+
+Individuals aren’t the only ones who can benefit from completing this quest -  organizations can too! Because earning this badge represents deep knowledge of an industry leading big data library, having the badge validates your organization’s understanding of Beam.. In addition, you can run the Beam library on a wide variety of runners, including Google Cloud Dataflow, Flink, Spark, and more,  making knowledge about this library highly transferable. Finally, your organization can use this quest as onboarding material for new hires on big data teams, allowing teams and organizations to get their newest employees up-to-date on the latest and greatest that Apache Beam has to offer.
```

Comment:
```suggestion
Individuals aren’t the only ones who can benefit from completing this quest -  organizations can too! Because earning this badge represents deep knowledge of an industry leading big data library, having the badge validates your organization’s understanding of Beam. In addition, you can run the Beam library on a wide variety of runners, including Google Cloud Dataflow, Flink, Spark, and more, making knowledge about this library highly transferable. Finally, your organization can use this quest as onboarding material for new hires on big data teams, allowing teams and organizations to get their newest employees up-to-date on the latest and greatest that Apache Beam has to offer.
```